ehrql/backends/tpp.py

```diff
@@ -1010,6 +1010,21 @@ def opa_proc(self):
         """
     )
 
+    ukrr = QueryTable(
+        """
+            SELECT
+                Patient_ID as patient_id,
+                dataset,
+                renal_centre,
+                creat as latest_creatinine,
+                eGFR_ckdepi as latest_egfr,
+                rrt_start as rrt_start_date,
+                mod_start as treatment_modality_start,
+                mod_prev as treatment_modality_prevalence
+            FROM UKRR
+        """
+    )
+
```

Comment:
If you don't have to do any transformation of the columns then you can get a simpler definition, and cleaner resulting SQL, by using `MappedTable` like we do here:
https://github.com/opensafely-core/ehrql/blob/3757e09de8ba397e426f78c53fb6a0dbe0df7fd6/ehrql/backends/tpp.py#L252-L260

Reply:
thanks, have done that